.github/workflows/database-copy.yml

```diff
@@ -1,18 +1,18 @@
-name: Copy PAAS databases to AKS
+# name: Copy PAAS databases to AKS
```

Comment:
The file should be deleted instead of commented out

Reply:
deleted